dask_saturn/core.py

```diff
@@ -68,6 +68,12 @@ class SaturnCluster(SpecCluster):
     """
 
     # pylint: disable=unused-argument,super-init-not-called,too-many-instance-attributes
+    def __await__(self):
```

Comment:
either this method should be added below `__init__()`, or you need to move that `pylint: disable` comment above this to `__init__()`. That comment is specific to `__init__()`.

Reply:
no reason to keep it above `__init__`.   will wait on feedback from julia after the break on the approach before making it better.